app/models/audit.rb

```diff
@@ -116,4 +116,11 @@ def validation_request
       sequence: activity_information
     )
   end
+
+  def valid_json?(comment)
+    JSON.parse(comment)
+    true
+  rescue JSON::ParserError, TypeError
+    false
+  end
```

Comment:
This could probably go in the helper, since it doesn't seem to relate to any of the instance variables.

Also I wonder if making it return the parsed JSON would make sense, that way we wouldn't have to parse it again in the view.